packages/block-library/src/navigation/index.php

```diff
@@ -191,11 +191,12 @@ function render_block_navigation( $content, $block ) {
  * @return string Returns  an HTML list from innerBlocks.
  */
 function core_block_navigation_build_html( $attributes, $block, $colors, $font_sizes ) {
-	$html            = '';
-	$classes         = array_merge(
+	$html    = '';
+	$classes = array_merge(
```

Comment:
Why the formatting change? These look like they were meant to be aligned with the ones below.

Reply:
I added a newline and then those were not aligned properly and linting complained. Rebase solved.